k8s/namespaces/fact/fact-admin/fact-admin.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: fact-frontend
```

Comment:
should these still be fact-frontend and not fact-admin?

Reply:
it shouldn't I updated it. This repo totally killed my atom